src/Containers/Impl/Docker/LocalDockerContainer.cs

```diff
@@ -14,9 +17,32 @@ public class LocalDockerContainer : IContainer {
 
         public bool IsShortId => Id.Length < 64;
 
-        public LocalDockerContainer(string id, string name) {
-            Id = id;
-            Name = name;
+        public IEnumerable<int> HostPorts { get; }
+
+        internal LocalDockerContainer(JToken containerObject) {
+            Id = (string)containerObject["Id"];
+            Name = GetContainerName(containerObject);
+            HostPorts = GetHostPorts(containerObject);
+        }
+
+        private IEnumerable<int> GetHostPorts(dynamic containerObject) {
+            var hostPorts = new List<int>();
+            try {
+                dynamic portMap = containerObject.NetworkSettings.Ports["5444/tcp"];
```

Comment:
No, you can have multiple containers running. The mapped host port will be different. for example:
Container1 5444/tcp -> 0.0.0.0:5444
Container2 5444/tcp -> 0.0.0.0:6464
Container3 5444/tcp -> 0.0.0.0:6897
and so on...
the 5444/tcp part remains constant for all containers that use the default configuration file. But on the container host (your computer) side they have to be on unique ports.